detection-bot/src/agent.ts

```diff
@@ -0,0 +1,31 @@
+import { Finding, HandleTransaction, TransactionEvent, FindingSeverity, FindingType } from "forta-agent";
+
+export const NETHERMIND_ADDRESS: { [address: string]: boolean } = {
+  "0x88dC3a2284FA62e0027d6D6B1fCfDd2141a143b8": true,
+};
+
+let findingsCount = 0;
+
+const handleTransaction: HandleTransaction = async (txEvent: TransactionEvent) => {
+  const findings: Finding[] = [];
+  const nethermindAddress = Object.keys(txEvent.addresses).find((address) => NETHERMIND_ADDRESS[address]);
```

Comment:
As it stands, this piece of logic is only registering when the Nethermind deployer address is involved in **any** transaction. Seeing as how the goal of the bot is for the specific transactions in which the Nethermind deployer address deploys a new bot when calling the correct Forta contract, the approach needs to be approached differently.

You are correct to think to use the `txEvent` the transaction handler will receive. To not give the answer away, I will let you know that the correct approach lies in this [_Transaction Event_ section](https://docs.forta.network/en/latest/sdk/#transactionevent) of the Forta docs, for your _"convenience"_. ;)

Seeing as how this will require a fundamental change to the bot, I will conclude my review at this point and can review more when this is "fixed".

Lastly, the bot should work with using the Nethermind deployer address assigned to a variable of type `string`.